install.sh

```
@@ -97,7 +97,56 @@ fi
+            fi
+        elif command -v wget >/dev/null 2>&1; then
+            if wget -qO "$FONT_FILE" "$FONT_URL" 2>/dev/null && wget -qO "$FONT_MONO" "$FONT_MONO_URL" 2>/dev/null; then
+                DOWNLOADED=true
+            fi
+        fi
+
+        if [ "$DOWNLOADED" = true ]; then
+            if command -v fc-cache >/dev/null 2>&1; then
+                fc-cache -f "$FONT_DIR" 2>/dev/null || true
+            fi
+            printf "${GREEN}✓${NC} Installed Nerd Font Symbols fallback font.\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)